pom.xml

```diff
@@ -75,7 +75,7 @@
         <okhttp.version>3.10.0</okhttp.version>
         <okio.bundle.version>1.14.0_1</okio.bundle.version>
         <okhttp.bundle.version>3.10.0_2</okhttp.bundle.version>
-        <opennms.api.version>0.1.0</opennms.api.version>
+        <opennms.api.version>0.4.1-SNAPSHOT</opennms.api.version>
```

Comment:
Can update to use release artifacts now.

Reply:
done.